src/mad_spark_alt/layers/quantitative/diversity.py

```
@@ -11,6 +11,11 @@
 from sentence_transformers import SentenceTransformer
 from sklearn.metrics.pairwise import cosine_similarity  # type: ignore
 
+from ...core.evaluation_utils import (
+    AsyncBatchProcessor,
```

Ruff (0.11.9)

`...core.evaluation_utils.AsyncBatchProcessor` imported but unused

Remove unused import: `...core.evaluation_utils.AsyncBatchProcessor`

(F401)